### PR TITLE
UML-1337: event codes renamed

### PR DESCRIPTION
**Pull request by SeemaMenon**
# Purpose

https://opgtransform.atlassian.net/browse/UML-1337

Fixes UML-1337

## Approach

_Explain how your code addresses the purpose of the change_

## Learning

_Any tips and tricks, blog posts or tools which helped you. Plus anything notable you've discovered about the Use a Lasting Power of Attorney service_

## Checklist

* [X] I have performed a self-review of my own code
* [ ] I have added relevant logging with appropriate levels to my code
* [ ] I have updated documentation (Confluence/GitHub wiki/tech debt doc) where relevant
* [ ] I have added tests to prove my work
* [ ] I have added welsh translation tags and updated translation files
* [ ] I have run an accessibility tool on any pages I have made changes to and fixed any issues found
* [ ] The product team have tested these changes

### CODE REVIEW

File: service-front/app/src/Common/src/Service/Log/EventCodes.php [hunk 1]
```diff
@@ -42,20 +42,26 @@ class EventCodes
...
     /**
      * LPA has activation key
      */
-    public const LPA_HAS_ACTIVATION_KEY = 'LPA_HAS_ACTIVATION_KEY';
+    public const OLDER_LPA_HAS_ACTIVATION_KEY = 'OLDER_LPA_HAS_ACTIVATION_KEY';
+
+    /**
+     * Older LPA match
+     */
+    public const OLDER_LPA_SUCCESS = 'OLDER_LPA_SUCCESS';
```
Comment (GemTay) [inline service-front/app/src/Common/src/Service/Log/EventCodes.php:65]: You just need to change the log message on line 81 in the AddOlderLpa class to say that a user has successfully added their older LPA, with the body containing the `'event_code' => $EventCodes::OLDER_LPA_SUCCESS`

Comment (SeemaMenon) [reply service-front/app/src/Common/src/Service/Log/EventCodes.php:65]: Did that Gemma... :), was a miss.

File: service-api/app/src/App/src/Service/Lpa/OlderLpaService.php [hunk 2]
```diff
@@ -9,6 +9,7 @@
 use App\Exception\ApiException;
 use App\Exception\BadRequestException;
 use App\Exception\NotFoundException;
+use Common\Service\Log\EventCodes;
```
Comment (GemTay) [inline service-api/app/src/App/src/Service/Lpa/OlderLpaService.php:12]: This use statement isnt needed here

Comment (SeemaMenon) [reply service-api/app/src/App/src/Service/Lpa/OlderLpaService.php]: removed

**Comment by SeemaMenon** [comment]
@GemTay  - request a review please :)

**Review by GemTay — APPROVED** [review]

**Pull request merged**